app/models/decidim/participatory_documents/zone.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryDocuments
+    class Zone < ApplicationRecord
+      include Decidim::Traceable
+      include Decidim::Loggable
+      include Decidim::TranslatableResource
+      include Decidim::TranslatableAttributes
+      include Decidim::Publicable
+      belongs_to :document, class_name: "Decidim::ParticipatoryDocuments::Document"
+      has_many :annotations, class_name: "Decidim::ParticipatoryDocuments::Annotation", dependent: :destroy
+
+      translatable_fields :title, :description
+      POSSIBLE_STATES = %w(draft published private closed).freeze
```

Comment:
I think we should have three different attributes, published or not (as stated by the `Decidim::Publicable`), private/not private and closed (with `closed_at` datetime attribute) .
In any case, is this used anywhere?

Reply:
@microstudi i have removed this so we can tackle in a separate PR.

Reply:
> Can we have combinations like: Published & private ? Private & closed?

Yes, we can have those combinations.